explorer/src/utils/nftoken.ts

```diff
@@ -0,0 +1,125 @@
+import axios from "axios";
```

Comment:
🚨 This doesn't look like it's an explicit dependency of Explorer. The only reason this probably works at all is because it's a dependency of `@metaplex/js`.

Reply:
Is it fine to add it as a dependency? Since it's already in the bundle...